### PR TITLE
5217-Remove-blueInk-to-introduce-Enlumineur

### LINKED ISSUE #5217: Remove blueInk to introduce Enlumineur
We can now remove the blueInk pretty printer 
And next introduce Enlumineur

### PR DESCRIPTION
**Pull request by Ducasse**
Fixes: #5217 remove blueink formatter and introduce the packages (but does not load for now enlumineur

### CODE REVIEW

**Comment by Ducasse** [comment]
I will fix the tests when introducing Enlumineur.

**Comment by MarcusDenker** [comment]
did you check if tests fail? The build seems to have failing tests after integrating this

File: src/BaselineOfBasicTools/BaselineOfBasicTools.class.st [hunk 1]
```diff
@@ -34,64 +34,67 @@ BaselineOfBasicTools >> baseline: spec [
...
-				baseline: 'Athens'
-					with: [ spec
-						loads: 'Cairo-core';
-						repository: repository ];
-				baseline: 'SUnit'
-					with: [ spec
-						loads: #('UI' 'Help' 'JenkinsSupport');
-						repository: repository ].
-			spec package: 'Tool-ExternalBrowser'.
-			spec package: 'BlueInk-Core'.
```
Comment (Ducasse) [inline src/BaselineOfBasicTools/BaselineOfBasicTools.class.st:59]: Restore these lines for issue #5376

File: src/BaselineOfIDE/BaselineOfIDE.class.st [hunk 2]
```diff
@@ -95,78 +95,90 @@ BaselineOfIDE >> additionalInitialization [
...
+			spec
+				repository: repository;
+				loads: 'development' ].
+		spec baseline: 'SUnit' with: [ 
+			spec
+				repository: repository;
+				loads: 'Tests' ].
+		"
+		here enlumineurFormatter
 		spec baseline: 'BlueInk' with: [
```
Comment (Ducasse) [inline src/BaselineOfIDE/BaselineOfIDE.class.st:129]: Restore for issue #5376

File: src/BaselineOfIDE/BaselineOfIDE.class.st [hunk 3]
```diff
@@ -95,78 +95,90 @@ BaselineOfIDE >> additionalInitialization [
...
 		spec package: 'System-Hashing-Testing'.
 		spec package: 'Rubric-SpecFindReplaceDialog'.
-		
 		spec baseline: 'Metacello' with: [ 
-			spec repository: repository; loads: #('Tests') ].
+			spec
+				repository: repository;
+				loads: #( 'Tests' ) ].
 		spec package: 'MonticelloGUI-Tests'.
-		spec baseline: 'BlueInk' with: [
```
Comment (Ducasse) [inline src/BaselineOfIDE/BaselineOfIDE.class.st:134]: Restore for issue #5376

**Pull request merged**